src/NzbDrone.Core/Sonarr.Core.csproj

```diff
@@ -4,7 +4,9 @@
     <Platforms>x86</Platforms>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="FluentMigrator.Runner" Version="1.6.2" />
+    <PackageReference Include="FluentMigrator" Version="4.0.0-alpha.268" />
+    <PackageReference Include="FluentMigrator.Runner" Version="4.0.0-alpha.268" />
+    <PackageReference Include="FluentMigrator.Runner.SQLite" Version="4.0.0-alpha.268" />
```

Comment:
I'm not sure but isn't referencing _only_ .SQLite enough? I assume it has dependencies to the other two.

Reply:
I had to add in the runner despite having the sqlite dependency so I fear it's required. But it's stupid so I'll double check just in case.

Reply:
It turned out you need `FluentMigrator.Runner` and `FluentMigrator.Runner.SQLite` but not `FluentMigrator`